types/swe-validation/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for swe-validation 1.0
+// Project: https://github.com/keype/swe-validation
+// Definitions by: Robert Hjalmers <https://github.com/hjalmers>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface cin {
+    isValid: boolean;
+    corporation: {
+        type: string;
+        id: string;
+    };
+}
+
+interface ssn {
+    isValid: boolean;
+    person?: {
+        type: string;
+        sex: string;
+        ssn: string;
+    };
+}
+
+declare let validate: {
```

Comment:
You don't need to wrap these functions in `validate`.
It would be best to follow the following template and simply export each of the functions and interfaces.
http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html

Reply:
As it stands now, the interface `ssn` is not accessible outside of this file.